cmd/command/list/list.go

```diff
@@ -16,12 +16,11 @@ package list
 import (
 	"github.com/spf13/viper"
 	"html/template"
-	"io"
 	"strings"
-	"text/tabwriter"
 
 	"github.com/edgexfoundry-holding/edgex-cli/config"
 	request "github.com/edgexfoundry-holding/edgex-cli/pkg"
+	"github.com/edgexfoundry-holding/edgex-cli/pkg/formater"
```

Comment:
s/formater/formatter
the spelling is too close

Reply:
I will remember it :)